templates/web/kingston/waste/garden/subscribe_confirm.html

```diff
@@ -1,4 +1,13 @@
-[% INCLUDE waste/header.html %]
+[%
+
+INCLUDE waste/header.html;
+
+IF report.title == 'Garden Subscription - Renew'; SET type = 'renew';
```

Comment:
Is there a way to share this 'type' logic across multiple templates? May not be worth doing right now, however.

Reply:
Yeah, I in fact got it from the email template in waste/_garden_data.html (which is how it's shared there between text/html) so perhaps could do similar here.